conf/locale/babel_third_party.cfg

```diff
@@ -4,3 +4,6 @@ input_encoding = utf-8
 
 [django: **/template/**.html]
 input_encoding = utf-8
+
```

Comment:
Wow, that's confusing that we use two tools called Babel now!

http://babel.pocoo.org/en/latest/
https://babeljs.io/